content/en/home/home_slider.md

```diff
@@ -8,7 +8,7 @@ item:
   cta_icon: fa fa-bullhorn
   cta_icon_pack: fas
   cta_label: Click here for more details
-  cta_url: /en/blog/release-v1.4
+  cta_url: /en/blog/release-v1.7
   overlay_color: '#555'
   overlay_filter: 0.25
   overlay_img: headers/header-kubecon.jpg
```

Comment:
The title of this home page slider  needs to be updated as well. 
`title: KubeEdge v1.4 is live!`

![image](https://user-images.githubusercontent.com/19162717/128152874-30bee5ee-a2ef-4de4-8bae-f1998a94147f.png)

Reply:
done